timeless/reservations/views.py

```diff
@@ -91,12 +96,15 @@ def edit(id):
     if request.method == "POST":
         flash("Edit not yet implemented")
     return render_template(
-        "restaurants/tables/create_edit.html", action="edit",
+        "reservations/create_edit.html", action="edit",
         id=id
     )
 
 
-@bp.route("/delete", methods=["POST"])
-def delete():
-    flash("Delete not yet implemented")
+@bp.route("/delete/<int:id>", methods=["POST"])
+def delete(id):
+    """ Delete reservation """
+    reservation = Reservation.query.get(id)
+    DB.session.delete(reservation)
+    DB.session.commit()
```

Comment:
@Vatavuk We agreed at some point to not tie the DB stuff to the views. We have now ``CrudAPIView`` -- can you use that to abstract the DB away? As I remember, it was used already in Reservation Settings.

Reply:
@amihaiemil this is a template view not an API view. I suppose it will be refactored using GenericView and/or DB abstraction (#288) but these things are not yet implemented.